svc-aggregation/agcommon/common.go

```diff
@@ -22,8 +22,19 @@ import (
 	"github.com/ODIM-Project/ODIM/lib-utilities/common"
 	"github.com/ODIM-Project/ODIM/lib-utilities/config"
 	"github.com/ODIM-Project/ODIM/svc-aggregation/agmodel"
+	uuid "github.com/satori/go.uuid"
 )
 
+// SupportedConnectionMethodTypes is for validating the connection method type
+var SupportedConnectionMethodTypes = map[string]bool{
+	"Redfish": true,
+	"SNMP":    true,
```

Comment:
set to false for everything other than redfish

Reply:
done

Reply:
done